packages/gatsby-plugin-gatsby-cloud/package.json

```diff
@@ -47,7 +50,9 @@
   },
   "sideEffects": false,
   "scripts": {
-    "build": "babel src --out-dir . --ignore \"**/__tests__\"",
+    "build": "babel src --out-dir . --ignore \"**/__tests__\" && yarn clean && yarn copy-type-declarations",
+    "clean": "yarn rimraf ./components/index.d.ts",
+    "copy-type-declarations": "yarn cpy src/components/index.d.ts components/index.d.ts",
```

Comment:
In other packages we use `npm run` instead of `yarn`.

I also think that for e.g. `rimraf` or `cpy` you can drop `yarn` anyways